src/site/assets/robots.txt

```diff
@@ -95,7 +95,6 @@ Disallow: /analytics-opt-out.html
 Disallow: /cgi-bin/
 Disallow: /scorecard
 Disallow: /drupal
-Disallow: /pittsburgh-health-care
```

Comment:
This change is actually part of https://github.com/department-of-veterans-affairs/va.gov-team/issues/5113, but it's so tiny and related I included it here.

Reply:
Turns out this PR can be merged immediately but this robots.txt change has to wait for launch day.